client/java/src/main/java/com/linecorp/centraldogma/client/Watcher.java

```diff
@@ -73,6 +94,34 @@
         }
     }
 
+    /**
+     * Waits for the initial value to be available and returns the specified default value if failed
+     * to retrieve the initial value from the server.
+     *
+     * @param timeout the maximum amount of time to wait for the initial value. Note that timeout is basically
+     *                a trade-off. If you specify a smaller timeout, this method will take less time even if
+     *                the server is not responsive, at the risk of falling back to the {@code defaultValue}.
```

Comment:
I'm not sure if it's your intention.
not responsive -> still responsive? (or just 'responsive'?)

Reply:
What I meant is: When a user specified a small timeout and the server is not responsive, a user will have a higher chance of getting `defaultValue`.

Reply:
Rephrased the sentence to make it clearer.